Module 2. Basics. Node.js fundamental theory./task2/index.js

```diff
@@ -0,0 +1,16 @@
+
+const csv = require('csvtojson');
+const fs = require('fs');
+const { pipeline } = require('stream');
+
+const inputFilePath = process.argv[2]; // input file from npm task argument
+const outputFilePath = process.argv[3]; // input file from npm task argument
+
+pipeline(fs.createReadStream(inputFilePath).pipe(csv()), fs.createWriteStream(outputFilePath),
```

Comment:
Are you sure this code will load the csv file line by line as suggested in evaluation criteria? (the file is not loaded fully in the RAM)

Reply:
yes, I think it's hint in our evaluation criteria, they recommend to use pipeline for this criteria.

Reply:
how are you going to prove that file isn't loading fully into RAM?